### ISSUE TITLE
order of fields: password comes before email

### ISSUE THREAD

**Issue by ap133** [pennersr/django-allauth#1461]
see: https://github.com/aellerton/demo-allauth-bootstrap/issues/16

Fix:

```
def set_form_field_order(form, fields_order):
    import collections
    if isinstance(form.fields, SortedDict):
        form.fields = collections.OrderedDict((f, form.fields[f])
                                  for f in fields_order)
    else:
        # Python 2.7+
        from collections import OrderedDict
        assert isinstance(form.fields, OrderedDict)
        form.fields = collections.OrderedDict((f, form.fields[f])
                                  for f in fields_order)
```

**Comment by pennersr** [comment]
Closed (94af1148a847b7634994ecf6d1f61addb712bf75)

**Comment by pennersr** [comment]
https://codeberg.org/allauth/django-allauth/issues/1461
